tests/test_manifester.py

```diff
@@ -226,6 +254,15 @@ def test_get_subscription_pools_with_offset():
     assert len(manifester.subscription_pools["body"]) > 50
 
 
+def test_subscription_allocation_username_prefix_filter():
+    """Test that all allocations in subscription_allocations property matching username prefix."""
+    manifester = Manifester(
+        manifest_category=manifest_data, requester=RhsmApiStub(in_dict=None, has_offset=True)
+    )
+    for allocation in manifester.subscription_allocations:
+        assert allocation["name"].startswith(manifest_data["username_prefix"])
```

Comment:
Is there a length of allocations we should also assert for to make sure it's returning the total you expect?

Reply:
I don't think so. One of the end goals of this effort is to provide users with a mechanism to clean up their existing subscription allocations, particularly ones of which they are unaware (e.g. allocations that were not cleaned up properly due to test interruption). So having an expectation on the number of allocations returned would be at cross purposes to that goal, in my thinking.